tqdm/_tqdm.py

```diff
@@ -15,7 +15,8 @@
 from ._utils import _supports_unicode, _environ_cols_wrapper, _range, _unich, \
     _term_move_up, _unicode, WeakSet
 import sys
-from time import time
+from threading import Thread, Event
+from time import time, sleep
 
 
 __author__ = {"github.com/": ["noamraph", "obiwanus", "kmike", "hadim",
```

Comment:
@casperdcl ordered the credits this way (chronology of arrival for each main contributor). What do you propose as an order?